.github/workflows/lra-test.yml

```diff
@@ -0,0 +1,38 @@
+name: LRA test
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - "rts/lra/**"
+  pull_request:
```

Comment:
@ochaloup I think we discussed to not have this on PR and just have it run on merged code for now. Is that your understanding?

Otherwise LGTM - thanks @xstefank